app/resources/results.html.template

```diff
@@ -91,8 +91,10 @@
       {% if results %}
         <h1>
           {% blocktrans %}
-            No results found for: "{{query_name}} {{query_location_original}}"
+            No results found for: "{{query}}"
```

Comment:
It looks inconsistent that the query is sometimes quoted (like this line) and sometimes not (line line 119). Can you clean them up to always quote the query?